roles/confluent.test/tasks/check_directory.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Check directory exists
+  stat:
+    path: "{{custom_path}}"
+  register: path_exists
+  
+- name: Assert path exists
+  assert:
+    that:
+       - path_exists.stat.exists
```

Comment:
I like this! Can you take it a step further and do assertions on ownership? heres an example we do in 600
https://github.com/confluentinc/cp-ansible/blob/6.0.x/roles/confluent.test/molecule/plaintext-rhel/verify.yml#L33

```
    - name: Get stats on copied file
      stat:
        path: /tmp/molecule.yml
      register: copied_file

    - name: Assert copied file's permissions correct
      assert:
        that:
          - copied_file.stat.exists
          - copied_file.stat.gr_name == 'confluent'
          - copied_file.stat.pw_name == 'cp-kafka'
        quiet: true
```